### PR TITLE
feat: add more workflow executor

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7311** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7311?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7231** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7231?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7159** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7159?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7123** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7123?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7119** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7119?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/src/plugins/copilot/workflow/service.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          );
+          throw new Error(`Edge ${edgeId} not found`);
+        }
+        node.addEdge(edge);
+      }
+    }
+    return workflow;
+  }
+
+  // todo: get workflow from database
```
Comment (forehalo) [inline packages/backend/server/src/plugins/copilot/workflow/service.ts]: todo(@darksky)

**Review by forehalo — COMMENTED** [review]
fix the linter complains

**Comment by darkskygit** [comment]
### Merge activity

* **Jun 25, 4:42 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/7231) rebased this pull request after merging its parent, because this pull request is set to merge when ready.
* **Jun 25, 4:53 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/7231) rebased this pull request after merging its parent, because this pull request is set to merge when ready.
* **Jun 25, 6:43 AM EDT**: `darkskygit` added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).
* **Jun 25, 7:05 AM EDT**: `darkskygit`  merged this pull request with the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).

**Review by forehalo — APPROVED** [review]

**Pull request merged**